scripts/create_ea_defs.sh

```diff
@@ -11,4 +11,4 @@ SSL_VERIFY="false"
 
 
 
-./create_ea_defs --grid-host=${GRID_HOST} --wapi-port=${WAPI_PORT} --wapi-username=${WAPI_USERNAME} --wapi-password=${WAPI_PASSWORD} --wapi-version=${WAPI_VERSION} --ssl-verify=${SSL_VERIFY}
+bin/create_ea_defs --grid-host=${GRID_HOST} --wapi-port=${WAPI_PORT} --wapi-username=${WAPI_USERNAME} --wapi-password=${WAPI_PASSWORD} --wapi-version=${WAPI_VERSION} --ssl-verify=${SSL_VERIFY}
```

Comment:
The bin directory is generated when the build happens and docker build puts the binary in the bin folder.

Anyways this will go way when the tools container image is ready.